frontend/src/components/shared/graphs/QuestionGraphView.jsx

```diff
@@ -25,7 +26,7 @@ function defaultNodePreProc(n) {
       y: -10,
     };
   }
-  if (n.curieEnabled) {
+  if ((Array.isArray(n.curie) && n.curie.length) || (n.curie && typeof n.curie === 'string')) {
```

Comment:
Curie should never be a string here. When we upload it the string is converted to an array. I tested it by uploading a query_graph with a string curie.